examples/simple_pendulum/run.py

```diff
@@ -39,3 +40,21 @@
 l = LagrangesMethod(L, [theta], forcelist=fl, frame=N)
 
 pprint(l.form_lagranges_equations())
+
+# Create a system from the lagranges equations
+sys = System(l)
+
+# Set up the system for simulation
+sys.times = linspace(0, 10, num=100)
+sys.constants = {m: 10, l: 5, g: 9.8}
+sys.initial_conditions = {theta: 60, thetad: 0}
+
+# Simulate the system
+out = sys.integrate()
+
+# Display the kinetic energy change in time (obtained from the particle in the
+# bodies list)
+KE = sys.body_kinetic_energies()
```

Comment:
I think that would be a good idea to have the same output. sys.integrate() seems to give the trajectories such that each column represents a single state. I feel this could be put to the same use here where each column represents a single body.